packages/server/src/cli.ts

```diff
@@ -171,8 +171,14 @@ if (isCli) {
 		const parts = [process.title];
 		parts.push(path.relative(process.cwd(), process.argv[1]));
 		for (let i = 2; i < process.argv.length; i++) {
-			if (process.argv[i].includes("--password")) {
-				parts.push(process.argv[i].replace(/=(.*)/, "=****"));
+			if (process.argv[i].includes("--password") || process.argv[i].includes("-P")) {
```

Comment:
I would change this... remove the `includes` and use something for equality. (`===`, maybe?).

If we somehow code-server is invoked by something like `code-server --foo "i-am-a-silly--password-usecase" --password "secret"`, we are going to have problems.

Reply:
True, that could happen. If we were to go the `===` route, we'd need to handle 4 different cases.

`--password`
`--password=`
`-P`
`-P=`

I figured the code looked and read cleaner in it's current state. And I figured the edge case of someone using `--password` in their filepath or other CLI args would be pretty rare and unlikely. And on that topic both approaches would break if a user wanted to run code-server in a directory named `--password`

Reply:
I agree. Let's leave the final word with the project owners then! 😉

Reply:
@code-asher 🙂

Reply:
Personally I'd prefer we properly handle all cases. It wouldn't even really be much more code, would it? 

Omitting the `-P` check for brevity:

```
if (process.argv[i].startsWith("--password=")) {
  parts.push(process.argv[i].replace(/=.*/, "=****");
} else if (process.argv[i] === "--password")) {
  parts.push(process.argv[i++], "****")
} else if (process.argv[i] === "--") {
  parts.push(...process.argv.slice(i));
  break;
} else {
  parts.push(process.argv[i]);
}
```

I think that might handle everything:
```
code-server --password=password            # =
code-server --password password            # space
code-server --password password My-Project # directory that contains the flag
code-server -- --password=                 # directory that is the same as or starts with the flag
code-server --password-max-tries           # a new flag we create that includes part of the password flag
```

Reply:
Ah! `--password-max-tries` is an example use case I hadn't even considered. Strict equality checks in this context would indeed make much more sense!

Reply:
Another note, as far as I remember `commander` requires the arguments to be exact, including capitalization, so checking against that isn't necessary (or desired).